api/Element.json

```diff
@@ -2667,7 +2667,7 @@
               "version_added": "10.1"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "3.1"
```

Comment:
After https://github.com/mdn/browser-compat-data/pull/17226 this is the only change that will remain of this PR. This change seems correct and is supported by source archeology:
https://github.com/WebKit/WebKit/commit/74c8babf4d4580c0a65ec8cda910c28d8bb9e35e
https://github.com/WebKit/WebKit/blob/74c8babf4d4580c0a65ec8cda910c28d8bb9e35e/WebCore/Configurations/Version.xcconfig

That's WebKit trunk version 525.1, mapping to Safari 3.1.